src/gui/transferlistdelegate.cpp

```diff
@@ -104,7 +104,7 @@ void TransferListDelegate::paint(QPainter *painter, const QStyleOptionViewItem &
         break;
     case TransferListModel::TR_UPSPEED:
     case TransferListModel::TR_DLSPEED: {
-            const qulonglong speed = index.data().toULongLong();
+            const qint64 speed = index.data().toLongLong();
```

Comment:
the correct type is `int`.

Reply:
Utils::Misc::friendlyUnit takes qint64.

Reply:
Ok and no warning.

Reply:
Guys, think again! Why is static_cast needed for convert from int to qint64 (aka long long int)?

Reply:
>Why is static_cast needed for convert from int to qint64 (aka long long int)?

Of course, it shouldn't be required. I'm just using it for showing how to do conversion.